src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super T> machines, T value);
```

Comment:
The `fill` method is correctly parameterized to accept a list of machines and a value of type `T`. Ensure that the implementation of this method in the implementing class handles different machine types as specified in the task.